### DISCUSSION TITLE
MIN ( AU , SPENDS , AUB )

### DISCUSSION THREAD

**Discussion by Freedom-kin** [kinfoundation/kin-rfcs#126] [_ Archive - KRE 3.1 Discussion]
If every AU has only 1 Kin in his wallet, then theoretically the AU would be equal to the AUB.

But the weighing of the AU is  x 0.1  and for the AUB it is  x 1.0

That means that the AUB will always be higher and therefore never the deciding factor.

What am I missing?

**Comment by Freedom-kin** [comment]
I think I found the answer to my question.
It's not about the AUB, it's about the Median Balance.
Mixed those two up.

**Reply by skyf0xx** [reply]
@Freedom-kin Please post any additional questions you have and we'll try to answer them.

**Comment by Freedom-kin** [comment]
This is from your proposal

AUmax, BalanceMax and SpendMax are the highest Active Users, highest Median Balance and highest Median spend attained by apps in that pay period.

How should I read that?
Is there one app that becomes the benchmark?

Or can the highest Active Users come from app 1
the Median Balance from app 2
and the Median Spends from app 3 ?
.

**Reply by skyf0xx** [reply]
@Freedom-kin This isn't from one app, it's from different apps that score the highest amount. This means one app can lead in Balances and another in Spends. Over time, apps compete upwards to have the highest economically feasible balances, and spends and AU
